strings/base_events.h

```diff
@@ -466,6 +495,24 @@ WINRT_EXPORT namespace winrt
             }
         }
 
+        void clear()
```

Comment:
I think just focusing on `clear` to support `IClosable` is a worthy effort. I'm reluctant to add more bells and whistles to `event` in general because there are just too many variations of event handling to satisfy everyone, but `clear` seems reasonably straightforward and universal.